### PR TITLE
testing/6tunnel: move to community / add runit example

### PR DESCRIPTION
**Pull request by itoffshore**
fixes whitespace in `6tunnel.c`
include `-u` option in help for dropping privileges
adds `runit` example script for service supervision

### CODE REVIEW

File: community/6tunnel/fix_help_add_runit.patch [hunk 1]
```diff
@@ -0,0 +1,394 @@
+diff --git a/6tunnel.c b/6tunnel.c
+index c33865a..b8d551e 100644
+--- a/6tunnel.c
++++ b/6tunnel.c
```
Comment (jirutka) [inline community/6tunnel/fix_help_add_runit.patch]: Trailing spaces bothers me too, but patching them in abuilds is quite pointless and undesirable…

**Comment by itoffshore** [comment]
The patch was merged upstream but the author has not made a new release.

**Comment by jirutka** [comment]
Aha, then it should be written in the patch, ideally with the original patch header.

**Comment by itoffshore** [comment]
the patch references the [PR](https://github.com/wojtekka/6tunnel/pull/5) - the patch itself is [https://github.com/wojtekka/6tunnel/pull/5.diff](https://github.com/wojtekka/6tunnel/pull/5.diff)

**Comment by jirutka** [comment]
Ah, I’ve already refactored it and pushed as 7bd9626a6e43b0670dfdaa70f0a8d5e1fbfc8b8a.

**Pull request closed without merging**